emr/application/controllers/Screeners.php

```diff
@@ -72,9 +72,9 @@ public function add(){
         $data["href"] = ["auth/logout"];
         $data["font"] = ["sign-out"];
 
-        $data["sideMenu"] = ["Calendar", "Availability"];
-        $data["link"] = ["main/index", "screeners/add"];
-        $data["icon"] = ["calendar","user"];
+        $data["sideMenu"] = ["Calendar", "Availability", "My Requests"];
+        $data["link"] = ["main/index", "screeners/add", "request/view"];
```

Comment:
why does the screener navigate to `request/view` to view requests but admin navigates to `insights`, they should be the same

Reply:
now admin navigates to `request/screener_requests`, why is it different? Both screener and admin should navigate too `request/view`

Reply:
Sorry about that, I overthought it. I changed it now so they both navigate to view